data/campaigns/The_Rise_Of_Wesnoth/units/Wesfolk_Leader.cfg

```diff
@@ -19,7 +19,7 @@
     {AMLA_DEFAULT}
     cost=60
     usage=mixed fighter
-    description= _ "Born in the Wesfolk aristocracy, she and her people were excluded from society by their lords, who betrayed their loyalty when a war against Haldric’s people was being lost. This outcast still conserves her nobleness in her veins and on the battlefield she has earned valuable experience, which continues increasing with time, as does her natural leadership."
+    description= _ "Born in the Wesfolk aristocracy, she and her people were excluded from society by their lords, who betrayed their loyalty when they started to lose a war against Haldric’s people. This outcast still conserves her nobleness in her veins, and on the battlefield she has earned valuable experience, which continues to increase with time, as does her natural leadership."
```

Comment:
Was the intention to make this change to the L2 Wesfolk_Lady too? The L2 and L3 previously had exactly the same text.

Reply:
Only the first change, "when they started to lose a war" has the intention to also make the change to also the Wesfolk Lady and Wesfolk Outcast. (A mistake that they were not all changed).
Regarding experience increasing with time, that is meant to be different.